src/HasEnums.php

```diff
@@ -162,10 +166,35 @@ protected function isEnumAttribute(string $key): bool
         return isset($this->enums[$key]);
     }
 
+    protected function isNullableEnum(string $key, $value): bool
```

Comment:
I would expect this method to return if the enum column is nullable at all and not if the value is null and allowed to be null.
This method mixes two checks.

Reply:
If the enum is nullable but not null it should definitely fallback on the enumerable check isn't it ?